src/components/Picklist/index.js

```diff
@@ -176,6 +183,9 @@ class Picklist extends Component {
         const { onChange } = this.props;
         const { activeOptionIndex } = this.state;
         const { label, name, icon, value } = this.activeChildren[activeOptionIndex];
+        this.setState({
+            selectedOptionName: name,
+        });
```

Comment:
we can't do this since there is two source of true, the only one that we need to use is value prop so there is not need to add other state for this